python/ray/_private/accelerators/amd_gpu.py

```diff
@@ -6,8 +6,22 @@
 
 logger = logging.getLogger(__name__)
 
-ROCR_VISIBLE_DEVICES_ENV_VAR = "ROCR_VISIBLE_DEVICES"
-NOSET_ROCR_VISIBLE_DEVICES_ENV_VAR = "RAY_EXPERIMENTAL_NOSET_ROCR_VISIBLE_DEVICES"
+if "ROCR_VISIBLE_DEVICES" in os.environ:
+    raise RuntimeError("Please use HIP_VISIBLE_DEVICES instead of ROCR_VISIBLE_DEVICES")
+
+if "HIP_VISIBLE_DEVICES" in os.environ:
+    hip_val = os.environ["HIP_VISIBLE_DEVICES"]
+    if cuda_val := os.environ.get("CUDA_VISIBLE_DEVICES", None):
+        logger.warning(
```

Comment:
Are they unequal?

Reply:
The line "assert hip_val == cuda_val" from the previous commit always firing even if they are equal. I used the logger warning as a workaround. Can you suggest a better way to handle this?

Reply:
Adding checks outside of functions makes it difficult to determine when they will be executed, especially since users might import packages in different orders.

Reply:
How about verifying it in `get_current_process_visible_accelerator_ids`?